tc-messaging/src/main/java/com/tc/entity/NetworkVoltronEntityMessageImpl.java

```diff
@@ -150,12 +159,21 @@ protected boolean hydrateValue(byte name) throws IOException {
     this.extendedData = getBytesArray();
     this.requiresReplication = getBooleanValue();
     this.oldestTransactionPending = new TransactionID(getLongValue());
+    
+    try {
+      if (this.type == Type.INVOKE_ACTION) {
+        MessageCodec<EntityMessage, EntityResponse> codec = supplier.getMessageCodec(this.entityDescriptor.getEntityID());
+        this.message = codec.decodeMessage(extendedData);
+      }
+    } catch (MessageCodecException exception) {
+//  swallow it
```

Comment:
That is right.  This is best efforts.  Another attempt is made to decode on the PTH were the client can be properly notified